### DISCUSSION TITLE
Date picker for Statistics graph card

### DISCUSSION THREAD

**Discussion by timvdsm** [home-assistant/frontend#20167] [Dashboard] [answered]
The Energy date picker (energy-date-selection card) is great when using the Energy cards. It allows you to define date ranges like day/week/month/custom range/... and easily navigate through them using the < and > buttons. 

It would be absolutely great to have the same functionality available for any Statistics graph card. 
So let's call it a Statistics date picker (statistics-date-selection card) for any Statistics graph card. 
That's something that is really missing now. 
Or even better, integrate this functionality directly in the Statistics card itself (as on option).

**Comment by karwosts** [comment, marked as answer]
Implemented in https://github.com/home-assistant/frontend/pull/23478

**Reply by timvdsm** [reply]
You are a hero! Many thanks! Really looking forward to it ;-)
